lib/asciidoctor/standoc/ref.rb

```diff
@@ -109,7 +111,7 @@ def fetch_ref(xml, code, year, **opts)
         return nil if hit.nil?
         xml.parent.add_child(Utils::smart_render_xml(hit))
         xml
-      rescue RelatonBib::RequestError
+      rescue Algolia::AlgoliaProtocolError, RelatonBib::RequestError
```

Comment:
We should catch the `Algolia` error inside the `relaton` gem, such as under `RelatonBib::ConnectionError`: `Metanorma::StanDoc` should not need to know about the `algolia` gem at all.

Reply:
Fixed

Reply:
Is this line now removed? We don't want any Algolia code in `metanorma-standoc`.